packages/build/README.md

```diff
@@ -101,6 +100,42 @@ Now you run the scripts, such as:
 npm run build
 ```
 
+5. Run code coverage reports
+- lb-nyc
+
+  By default, `lb-nyc` is a plain wrapper for https://github.com/istanbuljs/nyc
+
+  To customize the configuration:
+
+  - Create `.nycrc` in your project's root
+    directory
+
+    ```json
+    {
+      "include": [
+        "dist"
+      ],
+      "exclude": [
+        "dist/__tests__/"
+      ],
+      "extension": [
+        ".js",
+        ".ts"
+      ],
+      "reporter": [
+        "text",
+        "html"
+      ],
+      "exclude-after-remap": false
+    }
+    ```
+
+  - To run code coverage with every test execution update your package.json:
```

Comment:
> To run code coverage with every test execution update your package.json:

nitpick:

To run code coverage with every test execution, update your `package.json`: